appveyor/crowdinSync.py

```diff
@@ -0,0 +1,55 @@
+# based on file from https://github.com/jcsteh/osara
+
+import time
+import os
+import glob
+import sys
+import zipfile
+import shutil
+import subprocess
+import requests
+
+AUTH_TOKEN = os.getenv("crowdinAuthToken")
+if not AUTH_TOKEN:
+	raise ValueError("crowdinAuthToken environment variable not set")
+PROJECT_ID = int(os.getenv("crowdinProjectID"))
```

Comment:
Will raise a TypeError if not set

```suggestion
try:
    PROJECT_ID = int(os.getenv("crowdinProjectID", ""))
    if PROJECT_ID == 0:
        raise ValueError
except ValueError:
    raise ValueError("crowdinProjectID environment variable must be set and be a non-zero integer")
```